test/framework/shootframework.go

```diff
@@ -77,11 +77,11 @@ func NewShootFramework(cfg *ShootConfig) *ShootFramework {
 		Config:            cfg,
 	}
 
-	ginkgo.BeforeEach(func() {
+	CBeforeEach(func(ctx context.Context) {
 		f.CommonFramework.BeforeEach()
 		f.GardenerFramework.BeforeEach()
-		f.BeforeEach()
-	})
+		f.BeforeEach(ctx)
+	}, 2*time.Minute)
```

Comment:
Not sure `2mins` is OK here. As far as I can see, the old `ctx` had no explicit timeout; during `BeforeEach` it calls `f.AddShoot` which creates a seed client and e.g. has an own retry timeout of already `5mins` (should the seed be down temporarily or whatever).

Reply:
Agreed. I updated it to `8m`.